client/src/modules/cohort/model/selectors.js

```diff
@@ -36,5 +36,5 @@ export const getArchivedCohorts = createSelector(
 
 export const getMembers = createSelector(
   getCohortDetails,
-  cohort => cohort && _keyBy(cohort.members, '_id')
+  cohort => cohort && cohort.members
```

Comment:
this should be only: `cohort => cohort.members` if no members then empty object should be returned. Why checking if cohort exists?

Reply:
without this check we get an error: Uncaught TypeError: Cannot read property 'members' of undefined

Reply:
ok, let's leave it as it is for now - we will need to review cohort component and it's model/selectors